### ISSUE TITLE
bug: [Trainer controller] In trainer controller test cases, assert strings are too long

### ISSUE THREAD

**Issue by seshapad** [foundation-model-stack/fms-hf-tuning#187]
## Describe the bug

In the trainer controller test cases, assert string are long. They need to be wrapped. Here is an illustration:

Before change:
```
    assert (
        str(exception_handler.value).strip("'")
        == "Controller loss-controller-invalid-trigger has an invalid event (log_it_all_incorrect_trigger_name)"
```

After change:
```
    assert str(exception_handler.value).strip("'") == (
        "Controller loss-controller-invalid-trigger has"
        + " an invalid event (log_it_all_incorrect_trigger_name)"
```

## Platform

Please provide details about the environment you are using, including the following:

- Interpreter version:
- Library version: [X]

## Sample Code

NA

## Expected behavior

NA

## Observed behavior

NA

## Additional context

NA

**Comment by anhuong** [comment]
@seshapad can you provide an example of where this is occurring, how to fix, and why this needs to be fixed?

**Comment by seshapad** [comment]
> @seshapad can you provide an example of where this is occurring, how to fix, and why this needs to be fixed?

@anhuong I have modified the title of the issues and details to provide more details. Please review.

**Comment by seshapad** [comment]
This issue is addressed in PR [#169](https://github.com/foundation-model-stack/fms-hf-tuning/pull/169)
